sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/ConfigProperties.java

```diff
@@ -86,6 +88,56 @@ Double getDouble(String name) {
     }
   }
 
+  /**
+   * Get a duration property from the map, {@code null} if it cannot be found or it has a wrong
```

Comment:
Hmm this reminds me of the spec repo PR - in case you're not familiar, you need to press "Commit suggestion" to reflect it into the code, not only resolve the conversation.